<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>652</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>832</width>
    <height>652</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>832</width>
    <height>652</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Проект</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(235,222,207);</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>841</width>
      <height>631</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(235,222,207);</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Настройки</string>
     </attribute>
     <widget class="QPushButton" name="button_music">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>ON/OFF MUSIC</string>
      </property>
     </widget>
     <widget class="QPushButton" name="compendium">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Справочник команд</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_db_to_txt">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Сохранить базу данных
в .txt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_db_to_csv">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>70</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Сохранить базу данных
в CSV файл</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_add">
     <attribute name="title">
      <string>Добавить</string>
     </attribute>
     <widget class="QLabel" name="label1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Название:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Время:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ингредиенты:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Калории:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Рецепт:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="vvod1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>661</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="vvod2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>661</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="vvod3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>661</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="vvod4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>661</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="vvod5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>661</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_OK">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(173,255,47);</string>
      </property>
      <property name="text">
       <string>ОК</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="cmd">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>661</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="vvod0">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>661</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label0">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>661</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Номер (необязательно) (не пишите уже имеющийся в базе данных номер):</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_main">
     <attribute name="title">
      <string>Main</string>
     </attribute>
     <widget class="QTextEdit" name="zapros">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>501</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>811</width>
        <height>461</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_zapusk">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>10</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Запустить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_save">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>70</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_delete">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>70</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
